File: engine/engine-model/src/main/scala/net/server/channel/packet/command/admin/BlockPlayerCommandPacket.scala
package net.server.channel.packet.command.admin

class BlockPlayerCommandPacket(private var _mode: Byte, private var _victim: String, private var _type: Int, private var _duration: Int, private var _description: String) extends BaseAdminCommandPacket(_mode = _mode) {
  def victim: String = _victim

  def theType: Int = _type

  def duration: Int = _duration

  def description: String = _description
}

File: engine/engine-model/src/main/java/net/server/channel/packet/command/admin/BlockPlayerCommandPacket.java
package net.server.channel.packet.command.admin;

public class BlockPlayerCommandPacket extends BaseAdminCommandPacket {
   private final String victim;

   private final Integer theType;

   private final Integer duration;

   private final String description;

   public BlockPlayerCommandPacket(Byte mode, String victim, Integer theType, Integer duration, String description) {
      super(mode);
      this.victim = victim;
      this.theType = theType;
      this.duration = duration;
      this.description = description;
   }

   public String victim() {
      return victim;
   }

   public Integer theType() {
      return theType;
   }

   public Integer duration() {
      return duration;
   }

   public String description() {
      return description;
   }
}
